e2e/__tests__/multipleConfigs.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as path from 'path';
+import runJest from '../runJest';
+
+test('multiple configs will throw matching error', () => {
```

Comment:
we should also run the same test, but with `--config path/to/config` for both json and js and show that works

Reply:
Oh, so it is not a valid configuration, but you can still make it work if there is `--config` flag.

Ok.

Reply:
Done

Reply:
yeah, if explicitly passing `config` we read whatever you give us and skip any traversal (throwing if file doesn't exist or the config is invalid, but no attempts to red any _other_ file should be made)